exporter.go

```diff
@@ -81,10 +81,11 @@ func main() {
 		metricsPath = flag.String("telemetry.path", "/metrics", "URL path for surfacing collected metrics")
 
 		cephConfig = flag.String("ceph.config", "", "path to ceph config file")
+                cephUser = flag.String("ceph.user", "admin", "Ceph user to connect to cluster. The key file must be named ceph.client.<user>.keyring.")
```

Comment:
Also we can remove the trailing part of the flag usage i.e. `The key file must be named ceph.client.<user>.keyring.` since if the file is named something else but is included in the `keyfile` ceph config attribute that should work as well.